easybuild/easyblocks/o/openssl_wrapper.py

```diff
@@ -358,3 +359,90 @@ def sanity_check_step(self):
         custom_commands = ["ssl_ver=$(openssl version); [ ${ssl_ver:8:3} == '%s' ]" % self.majmin_version]
 
         super(Bundle, self).sanity_check_step(custom_paths=custom_paths, custom_commands=custom_commands)
+
+    def install_pc_files(self):
+        """Install pkg-config files for the wrapper"""
+
+        PC_TEMPLATE = """prefix=%(root)s
+exec_prefix=${prefix}
+libdir=${exec_prefix}/lib64
+includedir=${prefix}/include
+%(enginesdir)s
+
+Name: %(name)s
+Description: %(description)s
+Version: %(version)s
+%(requires)s
+%(libs)s
+%(cflags)s
+"""
+        openssl_components = {
+            'libcrypto': {
+                'name': 'OpenSSL-libcrypto',
+                'description': 'OpenSSL cryptography library',
+                'enginesdir': self.target_ssl_engine,
+            },
+            'libssl': {
+                'name': 'OpenSSL-libssl',
+                'description': 'Secure Sockets Layer and cryptography libraries',
+                'enginesdir': None,
+            },
+            'openssl': {
+                'name': 'OpenSSL',
+                'description': 'Secure Sockets Layer and cryptography libraries and tools',
+                'enginesdir': None,
+            },
+        }
+
+        pc_install_dir = os.path.join(self.installdir, 'lib64', 'pkgconfig')
+        mkdir(pc_install_dir)
+
+        for pc_comp in openssl_components:
+
+            pc_file = openssl_components[pc_comp]
+
+            # define component name in system pkg-config
+            pc_name = pc_comp
+            if self.majmin_version == '1.1':
+                # check suffixed names with v1.1
+                pc_name_suffix = pc_name + '11'
+                out, ec = run_cmd("pkg-config --exists %s" % pc_name_suffix, simple=False)
```

Comment:
So we have some type of non-fail flag here? I'm confused by the code. The code seems to use a global strictness parameter to determine whether or not to throw the error. The only way out i see if to turn off logging.

Reply:
Indeed, I disabled the hard error on a non-zero exit code with `log_ok=False, log_all=False`. That command does not generate any output in any case. Thanks @Micket
Fixed in https://github.com/easybuilders/easybuild-easyblocks/pull/2549/commits/70758dee4fcad6e09c4743203122b49f5b5ce9d2